clients/pkg/logentry/stages/metrics_test.go

```diff
@@ -321,8 +321,9 @@ func TestMetricStage_Process(t *testing.T) {
 			"contains_false":  "contains(keys(@),'nope')",
 		},
 	}
+	regexLogFixture := `11.11.11.11 - frank [25/Jan/2000:14:00:01 -0500] "GET /1986.js HTTP/1.1" 200 932ms"`
```

Comment:
Feels weird to me that we're redeclaring this, since there's a global one defined. Perhaps a different name, to be more clear?

Reply:
True, the problem is that the global `regexLogFixture` is used in other tests where the regex is different and the value after the HTTP status code is considered the HTTP response size. In this test, the same value was used to represent the request duration. Therefore, changing it in the global variable to contain the `ms` suffix would break the other tests. 
Will rename it to make it clearer 👍